### PR TITLE
Initial setup for the swerve modules

### PR DESCRIPTION
**Pull request by jkleiber**

### CODE REVIEW

**Comment by jkleiber** [comment]
@avidraccoon could you push your changes to this branch from yesterday please?

File: src/main/java/frc/robot/constants/SwerveConstants.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
 package frc.robot.constants;
 
 public final class SwerveConstants {
-    public static final double MAX_SPEED = 5.0; // m/s
-    public static final double MAX_ANGULAR_VELOCITY = 1.0; // rad/s
+    public static final double MAX_SPEED = 2; // m/s
```
Comment (aidnem) [inline src/main/java/frc/robot/constants/SwerveConstants.java:4]: This seems a little slow, I trust that it's for good reason?

**Review by aidnem — APPROVED** [review]
I trust that all of the tuning and offset values are good. If that's the case, looks good to me.

File: src/main/java/frc/robot/constants/SwerveConstants.java (hunk 1 above)
Comment (jkleiber) [reply src/main/java/frc/robot/constants/SwerveConstants.java:4]: I've probably been doing something wrong, but YAGSL seems to not care about the max speed. The drive seems open loop even though it's in closed loop mode 

I'm sure if we had more time / actually needed this we could figure it out

**Pull request merged**